mappings/net/minecraft/util/profiler/MetricSupplier.mapping

```diff
@@ -0,0 +1,2 @@
+CLASS net/minecraft/class_5952 net/minecraft/util/profiler/MetricSupplier
+	METHOD method_34705 getMetricSamplers ()Ljava/util/List;
```

Comment:
```suggestion
	METHOD method_34705 getSamplers ()Ljava/util/List;
```
can have this simpler name as this is already a metric supplier

Reply:
Would you also change the `MetricSampler` class to just `Sampler` though?

Reply:
Sure. Sampling is a good term for instantaneous sampling, so go for it.